rlberry/manager/agent_manager.py

```diff
@@ -1134,7 +1157,10 @@ def _optuna_objective(
         params_stats.fit()
 
         # Evaluate params
-        eval_value = np.mean(params_stats.eval_agents())
+        if not custom_eval_function:
```

Comment:
Seems that custom_eval_function is not covered by tests?

Reply:
fixed!